packages/just-scripts-utils/src/__tests__/mergePackageJson.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { mergePackageJson, _shouldUpdateDep } from '../mergePackageJson';
+import { IPackageJson } from '../IPackageJson';
+
+describe('_shouldUpdateDep', () => {
+  it('correctly detects deps needing update', () => {
+    // okay if old version is undefined
+    expect(_shouldUpdateDep(undefined, '0.0.1')).toBe(true);
```

Comment:
Can you split these into smaller `it()` test cases? It is relatively difficult to parse test failures this way.

Reply:
How small? Like I mentioned on another PR, Jest does do a good job of pointing out which line failed.

Reply:
Split somewhat. What do you think?

Reply:
great!